### PR TITLE
ft-adding redux

### PR DESCRIPTION
**Pull request by kareraolivier**
We Create the:
-React web-pack
-Add CSS 
-Also finished to add Redux

### CODE REVIEW

**Comment by kareraolivier** [comment]
Please review our PR we would use some feedback

File: src/Components/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react'
+
+
+function Home(){
+  return(
+    <div>
```
Comment (mansurissa) [inline src/Components/Home.jsx:5]: What is this `h1` for?

File: src/Components/Redux.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+			<div>Count: {props.count}</div>
+			<button onClick={() => props.increaseCounter()}>Increase Count</button>
+			<button onClick={() => props.decreaseCounter()}>Decrease Count</button>
+			<Outlet />
+		</div>
+	);
+}
+
+const mapStateToProps = (state) => {
+	console.log('state:', state);
```
Comment (mansurissa) [inline src/Components/Redux.jsx]: remove `console log`

File: src/redux/actioncreator.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import * as actions from './actiontypes.js'
+
```
Comment (mansurissa) [inline src/redux/actioncreator.js]: Please format your code. There are lots of empty lines

File: src/redux/actioncreator.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+    export const increaseCounter = () => {
+
+        return {
+
+            type: actions.INCREMENT,
+
```
Comment (mansurissa) [inline src/redux/actioncreator.js]: same here

File: src/redux/actioncreator.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        };
+
+    };
+
```
Comment (mansurissa) [inline src/redux/actioncreator.js]: same here (code formating)

**Review by mansurissa — CHANGES_REQUESTED** [review]
1. Work on comments I left
2. You and your team agree on a formating tool to use( I can suggest prettier and eslint) because there are lots of empty lines and we also need to have one empty line at the ed of each file

File: src/Components/Home.jsx (hunk 1 above)
Comment (capitale250) [reply src/Components/Home.jsx:5]: we were told to create a dummy route for the landing page

**Review by capitale250 — COMMENTED** [review]
thanks karera

**Review by mansurissa — APPROVED** [review]
Fine now

**Pull request merged**